watertap/property_models/activated_sludge/tests/test_new_modified_asm2d_reaction.py

```diff
@@ -0,0 +1,851 @@
+
+    @pytest.mark.unit
+    def test_unit_consistency(self, model):
+        assert_units_consistent(model) == 0
+
+    @pytest.mark.component
+    def test_solve(self, model):
+        model.fs.R1.initialize()
+
+        solver = get_solver()
+        results = solver.solve(model, tee=True)
+
+        assert check_optimal_termination(results)
+
+    @pytest.mark.component
+    def test_solution(self, model):
+        assert value(model.fs.R1.outlet.flow_vol[0]) == pytest.approx(0.21350, rel=1e-4)
+
+        assert value(model.fs.R1.outlet.temperature[0]) == pytest.approx(
+            298.15, rel=1e-4
+        )
+        assert value(model.fs.R1.outlet.pressure[0]) == pytest.approx(101325, rel=1e-4)
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_A"]) == pytest.approx(
+            23.01e-3, rel=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_F"]) == pytest.approx(
+            28.552e-3, rel=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_I"]) == pytest.approx(
+            30e-3, rel=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_N2"]) == pytest.approx(
+            0, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_NH4"]) == pytest.approx(
+            15.899e-3, rel=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_NO3"]) == pytest.approx(
+            0, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_O2"]) == pytest.approx(
+            0, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_PO4"]) == pytest.approx(
+            3.6168e-3, rel=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_K"]) == pytest.approx(
+            0, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_Mg"]) == pytest.approx(
+            0, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "S_IC"]) == pytest.approx(
+            4.830e-3, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "X_AUT"]) == pytest.approx(
+            0, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "X_H"]) == pytest.approx(
+            3.0e-2, rel=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "X_I"]) == pytest.approx(
+            25.0e-3, rel=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "X_PAO"]) == pytest.approx(
+            0, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "X_PHA"]) == pytest.approx(
+            0, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "X_PP"]) == pytest.approx(
+            0, abs=1e-4
+        )
+        assert value(model.fs.R1.outlet.conc_mass_comp[0, "X_S"]) == pytest.approx(
+            123.44e-3, rel=1e-4
+        )
+
+
+class TestAerobic15C:
```

Comment:
Does 15C correspond to something in the literature or SI material specifically?

Reply:
Not sure - just copied this test from the other ASM2d test files, which I believe are based on EXPOsan. Although the naming is a bit odd because the temperature in all of these tests is 25C, not 15C. So I'm curious where this 15C came from as well.